public/docs/ts/latest/glossary.jade

```diff
@@ -237,7 +238,8 @@ a#component
   ## Dependency injection
 .l-sub-section
   :marked
-    Dependency injection is both a design pattern and a mechanism
+
```

Comment:
I think this is your problem @juleskremer 
The whitespace after the `:marked` mixin is not allowed